02-tests/02-tests.md

```diff
@@ -2,11 +2,12 @@
 
 ### Usage
 
-Firstly, run the tapsetup tool in `dist/tools/tapsetup` if you want to test for
-`native`. Then, run the `RIOT/dist/tools/compile_and_test_for_board.py` script
+Firstly, run the tapsetup tool in `dist/tools/tapsetup/tapsetup` if you want
```

Comment:
The tool is not _in_ the path you changed it to. So either keep as is or remove the "in"

Reply:
Yup agreed.

Reply:
agreed